### PR TITLE
2 bugs fixed + Some improvements

### PR DESCRIPTION
**Pull request by odeimaiz**
##  What do these changes do?

Frontend:
- [x] Bug fixed: Side Panel's splitter works now from the beginning 
- [x] Responsive toolbar for NodesTree
- [x] Narrower Logger toolbar
- [x] Node data model: Node's status info taken to its new NodeStatus class

Backend:
- [x] Bug fixed: Remove template access rights when creating new instance

responsiveNodeTree

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: services/web/client/source/class/osparc/component/widget/NodesTree.js [hunk 1]
```diff
@@ -314,11 +329,31 @@ qx.Class.define("osparc.component.widget.NodesTree", {
...
+          toolBarBtns.forEach(toolBarBtn => {
+            const label = toolBarBtn.getChildControl("label");
+            label.exclude();
+            toolBarBtn.setToolTipText(label.getValue());
+          });
+        } else {
+          // Show Label
+          toolBarBtns.forEach(toolBarBtn => {
+            toolBarBtn.getChildControl("label").show();
+            toolBarBtn.setToolTipText(null);
```
Comment (GitHK) [inline services/web/client/source/class/osparc/component/widget/NodesTree.js:352]: After thinking about this it makes sense, no same tooltip when text is visible, is not useful.

File: services/web/client/source/class/osparc/data/model/NodeStatus.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/* ************************************************************************
+
+   osparc - the simcore frontend
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/data/model/NodeStatus.js:3]: MINOR: A bit more explanations about the rationale or original idea?

File: services/web/client/source/class/osparc/data/model/NodeStatus.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  },
+
+  properties: {
+    progress: {
+      check: "Number",
+      init: 0,
+      event: "changeProgress"
+    },
+
+    interactiveStatus: {
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/data/model/NodeStatus.js:36]: Is this some kind of status message? Line "pending", "running" ... ?

Comment (odeimaiz) [reply services/web/client/source/class/osparc/data/model/NodeStatus.js:36]: Yes, and depending on the text, for some cases, an icon is shown in the NodeUI

**Pull request merged**